mbuild/formats/xyz.py

```diff
@@ -66,6 +66,7 @@ def read_xyz(filename, compound=None):
                         "attribute.".format(name)
                     )
                     guessed_elements.add(name)
```

Comment:
Just out of curiosity, so we are not using this `guessed_elements` anywhere? I think it may be a helpful to warn/print this set out for the user.